api/end_device.proto

```diff
@@ -175,18 +176,30 @@ message EndDeviceVersion {
 }
 
 message MACSettings {
-  bool use_adr = 1 [(gogoproto.customname) = "UseADR"];
+  google.protobuf.BoolValue use_adr = 1 [(gogoproto.customname) = "UseADR"];
   // The ADR margin tells the network server how much margin it should add in ADR requests.
   // A bigger margin is less efficient, but gives a better chance of successful reception.
-  uint32 adr_margin = 2 [(gogoproto.customname) = "ADRMargin"];
+  google.protobuf.FloatValue adr_margin = 2 [(gogoproto.customname) = "ADRMargin"];
```

Comment:
Although this doesn't break existing JSON API, this is not binary compatible with our existing protos, so existing databases and existing clients/servers may become incompatible.

Reply:
We did not release the `3.0.0` yet, so non-backwards compatible changes should be fine